core/index.ts

```diff
@@ -20,7 +14,21 @@ function globalSession() {
   }
   return (global as any)._DF_SESSION as Session;
 }
-export const session = globalSession();
+const session = globalSession();
 
-// Used by generated index.js file.
-export const compileStandaloneSqlxQuery = compilers.compileStandaloneSqlxQuery;
+const supportedFeatures = {
+  arrayBufferIPC: true
+};
+
+// These exports constitute the public API of @dataform/core.
+// Changes to these will break @dataform/api, so take care!
+export {
+  adapters,
+  dataformCoreVersion,
+  indexFileGenerator,
+  compiler,
+  main,
+  compileStandaloneSqlxQuery,
+  session,
+  supportedFeatures
```

Comment:
optional, but suggest turning this into an array of enum values (potentially protobuf), i.e.

```
supportedFeatures: [ARRAY_BUFFER_IPC]
```